docs/content/concepts/assets/graph-backed-assets.mdx

```diff
@@ -93,18 +93,26 @@ def downstream_asset(middle_asset):
 
 ### Graph-backed multi-assets
 
-Using the <PyObject object="graph_multi_asset" decorator />, you can create a combined definition of multiple assets that are computed using the same graph of ops and same upstream assets.
+If you want to write a graph-backed asset that results in the materialization of multiple assets, you can use the <PyObject object="graph_multi_asset" decorator />.
```

Comment:
I found this section pretty confusing/hard to parse. I boiled it down to two reasons:
* the intro sentence was quite complicated, lots of noun soup with `combined definition of multiple assets that are computed using the same graph of ops and same upstream assets` 
* the example didn't explain what `two_outputs` did, nor did it really need to be a graph-backed multi asset since it was a single op (could have just been a plain multi asset)

So i played with the intro sentence a bit. This read easier to me, but that could just be a personal reading style thing, so very open to feedback.

I also updated the example to do some computation (even though that computation is trivial). This is a personal nit with some of our docs examples, but i feel like we often force the user to imagine/infer what the functions do, which can take away from understanding how the pieces fit together